### ISSUE TITLE
PyGMT can't load GMT library for Python 3.6 + Windows

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/pygmt#693]
**Description of the problem**

It seems PyGMT can't correctly find GMT for Python 3.6 + Windows. The error messages are:
```
pygmt.exceptions.GMTCLibNotFoundError: Error loading the GMT shared library 'gmt.dll, gmt_w64.dll, gmt_w32.dll, C:\Miniconda3\envs\test\Library\bin\gmt.dll':
```

It works before but failed in recent builds, for example, https://github.com/GenericMappingTools/pygmt/runs/1404103533.

**Comment by weiji14** [comment]
Is it:

1) The library is not there, because we broke something in https://github.com/conda-forge/gmt-feedstock/pull/111?
2) The library is there, but there is a problem linking to it, of which #628 might help?

**Comment by seisman** [comment]
See some debugging in #694. The gmt.dll is found in the system, but PyGMT can't load it.

**Comment by weiji14** [comment]
Closed with #699, see more discussion on why at #690.
